docs/source/installation.md

```diff
@@ -94,6 +105,8 @@ Once it's done, you can start to set up `vllm` and `vllm-ascend`.
 :selected:
 :sync: pip
 
+> NOTE: If you are installing vllm-ascend on an arch64 machine, The `-f https://download.pytorch.org/whl/torch/` command parameter in this section can be omitted. It's only used for find torch package on x86 machine.
```

Comment:
I remove this intentionally, seems a little complex in here, I think it's fine without it.